flexmeasures/ui/static/openapi-specs.json

```diff
@@ -1,4 +1,18 @@
 {
+  "info": {
+    "description": "Welcome to the NICs EMS API.<br/>This is a platform for smart energy scheduling (a \"Cloud EMS\").<br/><br/>You can try out all these endpoints if you are authenticated in the platform.Or you can paste your API key token with the button on the right.<br/><br/>To authenticate your code, include an `Authorization` header with your API key.Also, check out the <a href=\"https://github.com/FlexMeasures/flexmeasures-client\" target=\"_blank\">Python client</a>.",
```

Comment:
This now became NICs EMS. I guess the docs need to be generated with the new settings set to some default in the FM config used when creating docs.